app/tpl/result.hbs

```diff
@@ -8,7 +8,7 @@
                 <a href="/" class="gh-btn gh-btn-blue"><span>Start Over</span></a>
             </h1>
             <div class="health-meter {{results.score.level}}">
-                <h4>Theme compatibility score</h4>
+                <h4>Theme compatibility for Ghost version {{checkedVersion}}</h4>
```

Comment:
@AileenCGN do you think this makes more sense?

Reply:
I think so. Because the user has to select the version that they want to test it again on the starting screen. IMO this would e. g. flag up if they selected a wrong version unintentionally.